### DISCUSSION TITLE
Error while using OpenCV

### DISCUSSION THREAD

**Discussion by Traplekumar** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#75] [4.0 Exploring the data and building models]
Hello everyone, 
I am getting an error while using selectROI function in OpenCV. I tried to search it on the internet but solutions suggested that I should rebuild the system, I don't know how to do this in Azure Platform. I am using Azure and Windows10 OS.
Does anyone know to solve this?
![error](https://user-images.githubusercontent.com/64093713/120234171-0d989e00-c275-11eb-81c4-3327990b8572.png)

Error- OpenCV(4.5.2) /tmp/pip-req-build-9qn4uc85/opencv/modules/highgui/src/window.cpp:679: error: (-2:Unspecified error) The function is not implemented. Rebuild the library with Windows, GTK+ 2.x or Cocoa support. If you are on Ubuntu or Debian, install libgtk2.0-dev and pkg-config, then re-run cmake or configure script in function 'cvShowImage'

**Comment by EY-glowglowglow** [comment]
@codeindulgence any ideas? :D

**Reply by nickbutler** [reply]
Not for this specific error in a Windows environment, I'm afraid.
I think other participants have used OpenCV in our [template environment](https://github.com/EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge/wiki/1b.-Getting-started:-Azure-Environment), so maybe try using that as recommended.
Otherwise this problem is specific to [OpenCV on Windows](https://docs.opencv.org/master/d3/d52/tutorial_windows_install.html).
